tests/cases/standalone/common/alter/change_fulltext_options.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE `test` (
+  `message` STRING FULLTEXT,
+  `time` TIMESTAMP TIME INDEX,
+) WITH (
+  append_mode = 'true'
+);
+
+SHOW CREATE TABLE test;
+
+ALTER TABLE test MODIFY COLUMN message SET FULLTEXT WITH(analyzer = 'Chinese', case_sensitive = 'true');
+
+SHOW CREATE TABLE test;
+
+ALTER TABLE test MODIFY COLUMN message SET FULLTEXT WITH(analyzer = 'Chinese', case_sensitive = 'false');
```

Comment:
Validate the option modification after the system restart.

e.g., 
```sql
-- SQLNESS ARG restart=true
SHOW CREATE TABLE test;
```

However, the `SHOW CREATE TABLE` only can validate the metadata of the table. Do we have a way to validate the options at the engine level? @zhongzc @evenyag